share/actionsmap.yml

```diff
@@ -2082,3 +2082,15 @@ diagnosis:
                     help: Remove a filter (it should be an existing filter as listed with "ignore --list")
                     nargs: "*"
                     metavar: CRITERIA
+
+
+#############################
+#           Storage         #
+#############################
+storage:
+  category_help: Manage hard-disks, filesystem, pools
+  actions:
+    # storage_disks_list
+    disks_infos:
```

Comment:
Is it possible to declare nested subcomands? Like `yunohohost storage disk infos`?

Reply:
yes, have a look at the "cert" subcategory inside "domain"